packages/contact-center/cc-widgets/.babelrc.js

```diff
@@ -0,0 +1,3 @@
+const baseConfig = require('../../../.babelrc');
```

Comment:
sorry to ask it again , i remember asking this , do we need a babel because we are consuming sdk or its for something else

Reply:
Jest is what is using babel config. When the tests are written in typescript I think thats when we require babel.